### DISCUSSION TITLE
Set prefix for conditioning (prompt) for API use

### DISCUSSION THREAD

**Discussion by Szaamaan** [comfyanonymous/ComfyUI#5971] [Q&A] [answered]
I'm not sure if this is a problem which I need to tackle from the ComfyUI side or from my current AI chat driver (Open WebUI). I've got a (relatively) decent workflow for basic queries that generates acceptable results, and I'd like to use this workflow for image generation from Open WebUI.

The negative prompt I can leave as is and it becomes "hardcoded". But I need a "prefix" of elements for the positive prompt to get adequate results and here I'm not sure what to do.

For testing I picked some fixed seeds and played around with the prompt. In general, I'd expect to achieve the same results with the prompt "[my list of prefixed quality prompts] [actual prompt]" and when using a concatenation of "[my list of prefixed quality prompts]" and "[actual prompt]"... but I got different results.

Any suggestions on getting what I'm trying to achieve?

**Comment by ltdrdata** [comment]
How did you concatenate 2 prompts?

**Reply by Szaamaan** [reply]
I tried using the two "combine" or "concat" conditioning nodes available in ComfyUI.

So in effect it's 
```
MODEL -> TEXT #1 -> one of the combine nodes mentioned -> rest of the workflow
      \> TEXT #2 /
```
I should probably mention that the results weren't "bad" bad - with a decent model it's sometimes hard to tell if the quality prompt components are being used or not but what did throw me off was that the results were always different from the "base", single text prompt flow result.

**Comment by Szaamaan** [comment, marked as answer]
I found a solution. Rather than trying to combine the default conditioning nodes I now use basic string input nodes with a string combine node. This feels a lot more fitting to what I am trying to do and seems to work as expected.

The combined string is then fed to the "standard" prompt node (the string can be linked to, despite there being no node point initially).

**Comment by ltdrdata** [comment]
The "conditioning combine", "conditioning concat," and "string concat" each have different meanings.
